main/src/menu.ts

```diff
@@ -38,12 +38,8 @@ export const buildMenu = (mainWindow, loadInit) => {
         })
     }
 
-    const executeVimCommandForMultipleFiles = (
-        browserWindow: BrowserWindow,
-        command: string,
-        files: string[],
-    ) => {
-        executeMenuAction(browserWindow, { evt: "open-files", cmd: [command, files] })
+    const openMultipleFiles = (browserWindow: BrowserWindow, files: string[]) => {
```

Comment:
Command wasn't needed here since the new function takes the users config into account.
Also renamed it since `executeVimCommandForMultipleFiles` that only called `"open-files"` felt misleading.